src/python/dxpy/scripts/dx.py

```diff
@@ -2622,6 +2622,18 @@ def list_developers(args):
     except:
         err_exit()
 
+def list_files(args):
+    try:
+        results = dxpy.api.database_show_files(
```

Comment:
Was there a corresponding back end PR for this?

Reply:
https://github.com/dnanexus/nucleus/pull/2632